dashboard/app/models/pd/application/teacher_application.rb

```diff
@@ -262,20 +262,6 @@ def principal_approval_url
       pd_application_principal_approval_url(application_guid) if application_guid
     end
 
-    def principal_approval_state
```

Comment:
This was being overwritten in a different principal_approval_state method 🤯 ...

Reply:
A principal approval is only created when it's submitted –– the link to it doesn't create one on the backend. So for example, on staging, there's a link to a principal approval here https://staging-studio.code.org/pd/application_dashboard/csa_teachers/11 but the record doesn't exist until it's been submitted.

So "Complete" only refers to the principal approval being submitted (i.e. completed), but the approval itself depends on the :do_you_approve part